MediaGalleryUi/view/adminhtml/web/css/source/_module.less

```diff
@@ -368,3 +368,21 @@
         margin-right: 1.4rem;
     }
 }
+
+.media-width(@extremum, @break) when (@extremum = 'min') and (@break = @screen__m) {
+    .media-gallery-image-details-modal {
+        .image-details {
+            .lib-vendor-prefix-display();
+
+            .image-details-sidebar {
+                .lib-vendor-prefix-flex-grow(1);
+                margin-top: 0;
+                padding-left: 40px;
+            }
+
+            .image-details-image img {
+                max-height: 650px;
+            }
```

Comment:
I believe we can remove this and set it on `.image-details img` outside the media query, what do you think?